api/graphql/schema.graphql

```diff
@@ -3219,6 +3232,10 @@ type Mutation {
     withROD: Boolean
   ): Boolean! @guard
   downloadUsersCsv(ids: [UUID!], where: UserFilterInput): Boolean! @guard
+  downloadCommunityInterestsCsv(
+    ids: [UUID!]
+    where: CommunityInterestFilterInput
+  ): Boolean! @guard
```

Comment:
A small bit of naming pedantism
Given this is downloading users through community interest relations and not downloading community interest records, I think a name like `downloadCommunityInterestUsersCsv` will be more specific. Just in case we want to download the community interests themselves someday

Would need to adjust a few other files names too

Reply:
Renamed in [296f3af](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13012/commits/296f3af709b8970e687ae10b947abdeca06af39c)